src/helpers/typesciptCssModulesHelper.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Return style by key - solves to fix noImplicitAny errors when referencing modular styles from ts files via index accessor
+ * @param styles modular styles object
+ * @param key string classname
+ */
+export function getStyle<StylesType>(styles: StylesType, key: string): StylesType[keyof StylesType] | undefined {
```

Comment:
TS is really good at inferring types:

```suggestion
export function getStyle<StylesType>(styles: StylesType, key: string) {
```

Reply:
I know, I've just left it for clarity) But let's remove it if u think that would be better